code/game/objects/item.dm

```diff
@@ -700,19 +700,26 @@ var/global/list/slot_flags_enumeration = list(
 		blood_overlay.color = COLOR_LUMINOL
 		update_icon()
 
-/obj/item/add_blood(mob/living/carbon/human/M, amount = 2, blood_data)
+/obj/item/add_blood(mob/living/carbon/human/human, amount = 2, blood_data)
```

Comment:
Why change the arg name? This is now different to every other add_blood proc.

Reply:
a more descriptive name for a variable than just a single letter, I guess

Reply:
> Why change the arg name? This is now different to every other add_blood proc.

a more convenient name for an argument than just one letter, and in a good way there should not be arguments consisting of one letter at all

Reply:
There are stupid byond issues with locally renaming proc arguments. Either rename it everywhere or don't rename it.